lib/templateFactory.js

```diff
@@ -265,13 +266,16 @@ class TemplateFactory extends BaseFactory {
             const templateIds = templates.map(t => t.id);
 
             const metricJobsListConfig = {
-                params: { templateId: templateIds },
+                params: { templateId: templateIds, archived: false },
                 readOnly: true,
                 aggregationField: 'templateId'
             };
 
             const metricBuildsListConfig = { ...metricJobsListConfig };
 
+            // archived is not an attribute of build, so we need to filter it out
+            // builds are historical data so no need to filter out builds from archived jobs
+            metricBuildsListConfig.params = { templateId: templateIds };
```

Comment:
Since `metricBuildsListConfig` is not inheriting any significant config from `metricJobsListConfig`, let define it directly.
```suggestion
            // builds are historical data so no need to filter out builds from archived jobs
            const metricBuildsListConfig =  {
                params: { templateId: templateIds },
                readOnly: true,
                aggregationField: 'templateId'
            };;
```

Reply:
since this is trivial, I decided to ignore it for now and merged the PR. Thank you for the input though.